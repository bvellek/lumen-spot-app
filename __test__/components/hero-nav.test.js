import React from 'react';
import renderer from 'react-test-renderer';

import HeroNav from '../../public/js/components/hero-nav';

describe('<HeroNav />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <HeroNav />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
