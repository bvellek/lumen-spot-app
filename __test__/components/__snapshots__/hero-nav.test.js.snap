// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroNav /> it renders correctly 1`] = `
<nav
  className="landing-nav"
>
  <a
    className="sliding-middle-out"
    onClick={[Function]}
    style={Object {}}
  >
    Search...
    <span
      className="visually-hidden"
    >
      for a location to find sun, weather, and image results
    </span>
  </a>
</nav>
`;
